gulpfile.js

```diff
@@ -245,3 +258,73 @@ gulp.task('umd-watch', function () {
     gulp.watch(PATHS.ADMIN_JAVASCRIPT_SRC + '/*.js', ['umd-admin']);
     gulp.watch(PATHS.FRAMEWORK_JAVASCRIPT_SRC + '/*.js', ['umd-framework']);
 });
+
+
+/*
+ * Sprite and scss/css compilation
+ */
+ 
+gulp.task('compile', ['sprites', 'compile-admin:css', 'compile:css', 'compile-dev-install:css'], function () {
```

Comment:
`compile` isn't a great name here since we already have `build`, which should really generate everything (JS, CSS and images). Suggest we rename the task to `css`, with "subtasks" `sprites` and `css`. I don't think there's a need to separate out tasks for `framework/scss`, `framework/admin/scss` and `framework/dev/install/scss`, unless I'm missing something here?